sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/ContainerRegistryChartPushedEventData.java

```diff
@@ -4,11 +4,11 @@
 
 package com.azure.messaging.eventgrid.systemevents;
 
-import com.azure.core.annotation.Immutable;
+import com.azure.core.annotation.Fluent;
 import java.time.OffsetDateTime;
 
 /** Schema of the Data property of an EventGridEvent for a Microsoft.ContainerRegistry.ChartPushed event. */
-@Immutable
```

Comment:
Is those Immutable changes to Fluent intentional?

Reply:
Yes. These types weren't actually Immutable anyway - they have setters. This was a bugfix in autorest.java.